src/container/internal.tsx

```diff
@@ -109,50 +116,65 @@ export default function InternalContainer({
         styles.root,
         styles[`variant-${variant}`],
         fitHeight && styles['fit-height'],
+        media?.content && (mediaPosition === 'top' ? styles['with-top-media'] : styles['with-side-media']),
```

Comment:
minor: I think I've counted `media?.content` five times here. Might be worth doing `const hasMedia = !!media?.content`.

That also would turn it into a proper boolean, and not `boolean | undefined`

Reply:
Nice, I updated it. And just like that... way cleaner!

Thanks!